schema/definitions/0.8.0/schema/fmu_meta.json

```diff
@@ -352,6 +352,25 @@
       "title": "CPGridSpecification",
       "type": "object"
     },
+    "Context": {
+      "description": "The internal FMU context in which this data object was produced",
+      "properties": {
+        "stage": {
+          "enum": [
+            "case",
+            "iteration",
+            "realization"
```

Comment:
Note to self and PO: Need to discuss of these are the complete stages; e.g. we have also have a "preprocessed" as an option, and more? (the ever returning boomerang on definitions...)

Reply:
Moved to another PR: https://github.com/equinor/fmu-dataio/pull/460